tests/relay_integration/lang/java/test_plugin.py

```diff
@@ -84,6 +84,12 @@ class AdditionalInnerClass {
             throw RuntimeException("whoops")
         }
     }
+
+    class OneMoreInnerClass {
+        fun whoops4() {
```

Comment:
Not a big deal but this is never called anywhere in the code sample. Should we call it from `whoops3` above? Could be confusing for the next person coming along and trying to make sense of the example(s).